README.md

```diff
@@ -54,38 +55,38 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - run: perl -V
+      # adjust that section to fit your distribution
+      #   cpanm is available at that stage
+      - run: cpanm install .
```

Comment:
it may be better to promote https://github.com/perl-actions/ci-perl-tester-helpers/ as well ...

Reply:
o yes you are right this can be confusing, and these extra helpers can be super useful there
note that `perldocker/perl-tester` provides for free the `cpanm` client, this is why I was using it there
I wonder if `ci-perl-tester-helpers` should provide a more standard `v1` tag

Reply:
I've added the ci helpers bundle

Reply:
ci-perl-test-helpers can be  also used as actions